internal/nonkube/bootstrap/controller/systemd.go

```diff
@@ -130,7 +130,7 @@ func (s *systemdServiceInfo) GetServiceFile() string {
 	if s.GetUid() == 0 {
 		return path.Join(s.rootSystemdBasePath, s.GetServiceName())
 	}
-	return path.Join(api.GetSystemConfigHome(), "systemd/user", s.GetServiceName())
+	return path.Join(api.GetHostDataHome(), "systemd/user", s.GetServiceName())
```

Comment:
I believe that this function should return something similar to what `internal/nonkube/common/systemd.go` returns on rootless cases. Example:

```
	if s.GetUid() == 0 {
		return path.Join(s.rootSystemdBasePath, s.GetServiceName())
	}
	return path.Join(api.GetConfigHome(), "systemd/user", s.GetServiceName())
```

As the intention is to get the config home to place the `.service` file.
And this won't be useful if the `install` command itself is running in a container.